### DISCUSSION TITLE
Updating parameters min-max interval given a certain input

### DISCUSSION THREAD

**Discussion by f-iniv** [pymeasure/pymeasure#472] [Q&A] [answered]
Hi, I have a Razorbill RP100 power supply, and one of the devices it controls has a temperature x voltage curve where it limits its minimum and maximum voltages, so I created a FloatParameter to control that, where I added its min and max as variables following the temperature curve, and the temperature as an IntegerParameter, but the interval is only updated on starting the program with the default set in the temperature. Is there a way for the min-max interval to be updated every time I change the temperature value?

**Comment by CasperSchippers** [comment, marked as answer]
If I understand correctly, you have a FloatParameter of which you'd want to change the limits you enter in the `values` argument. Currently, there are two PR's open to make this a variable property rather than a static one as it currently is, see #250, #371 and #376.

**Reply by f-iniv** [reply]
Yes, that seems to be precisely what I need. Thank you!
